src/Benchotron/UI/Console.purs

```diff
@@ -94,9 +93,9 @@ runBenchmarkConsole benchmark = do
   lift $ do
     stderrWrite "\n"
     noteTime \t -> "Finished at: " <> t <> "\n"
-  return r
+  pure r
   where
-  noteTime f = now >>= toLocaleTimeString >>= (stderrWrite <<< f)
+  noteTime f = now >>= show >>= (stderrWrite <<< f)
```

Comment:
I'd rather not use `Show` if we can avoid it here. Is there no longer a function that corrresponds to what `toLocaleTimeString` used to do?